Frontend Visual Update

JackMcPhillips1543 opened the pull request:
Fixes #64 

What was changed:
The frontend visuals for the treadmill view were all updated and revamped.

Why was it changed:
It was changed because the client wanted it to look better and make what was going on more obvious to the patient.

How was it changed: 
Current steptime was changed from an X for all steps to a green circle if the step was in range and a red X if it was out of range. A box highlights the range instead of just 2 lines, the background color was changed, and a baseline was setup

git-voo reviewed (changes requested):
Good job Jack. The general structure of your output looks great. It gives us something to tweak as the requirements take further shape. Just some minimal adjustments and we are good to go.

1. The keys "Left" and "Right" should be outside the frame of the treadmill such that the target zone NEVER gets to that point. Remember, if the strong vertical line represents the range of movements from 0-100 of some metric,  then the keys are placed somewhere around -5 of the same metric.

![image](https://github.com/user-attachments/assets/1d784979-4def-4eac-9ef9-a9a40b356aec)

2. The patient seem to be stagnant on the median line while the target zones are changing. I understand that this may be as a result of our present data/formula, we need to take note of this phenomenon and test for it when our data/formula is updated. If possible, use random data to test and ensure the patient moves along the y axis just like the target zones.

JackMcPhillips1543 commented:
> Good job Jack. The general structure of your output looks great. It gives us something to tweak as the requirements take further shape. Just some minimal adjustments and we are good to go.
> 
> 1. The keys "Left" and "Right" should be outside the frame of the treadmill such that the target zone NEVER gets to that point. Remember, if the strong vertical line represents the range of movements from 0-100 of some metric,  then the keys are placed somewhere around -5 of the same metric.
> 
> ![image](https://private-user-images.githubusercontent.com/111019083/414509922-1d784979-4def-4eac-9ef9-a9a40b356aec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk5MzM3MzcsIm5iZiI6MTczOTkzMzQzNywicGF0aCI6Ii8xMTEwMTkwODMvNDE0NTA5OTIyLTFkNzg0OTc5LTRkZWYtNGVhYy05ZWY5LWE5YTQwYjM1NmFlYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxOVQwMjUwMzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNmI2OGQ4MTE4NjUzYzAzOTQxMDU1MmJhNjdlZTBmMzQxNTc3NzNjMmM1NTIwZWY2MzNmODExNTJlZGNhN2UwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.zGT7GjJWA-jcBGt0LrQYuCfTxRZg0c18_hpCaacP7zo)
> 
> 2. The patient seem to be stagnant on the median line while the target zones are changing. I understand that this may be as a result of our present data/formula, we need to take note of this phenomenon and test for it when our data/formula is updated. If possible, use random data to test and ensure the patient moves along the y axis just like the target zones.

To address this issues I moved the left and right text outside completely so they will never touch, I also added random numbers so you can see how the the current steps move.
